### PR TITLE
feat: add pod disruption budgets view

### PR DESCRIPTION
**Pull request by TristanHoladay**
## Description
Add api and view for pod disruption budgets

## Related Issue

- #48

### CODE REVIEW

File: ui/src/lib/features/k8s/cluster-ops/pod-disruption-budgets/component.svelte [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- SPDX-FileCopyrightText: 2024-Present The UDS Authors -->
+
+<script lang="ts">
+  import { DataTable } from '$components'
+  import { createStore, type Columns } from './store'
+
+  export let columns: Columns = [
+    ['name', 'emphasize'],
+    ['min_available'],
```
Comment (decleaver) [inline ui/src/lib/features/k8s/cluster-ops/pod-disruption-budgets/component.svelte:10]: just missing namespace

**Comment by TristanHoladay** [comment]
Screenshot from 2024-07-26 08-43-49

File: pkg/api/resources/cache.go [hunk 2]
```diff
@@ -16,6 +16,7 @@ import (
 	batchV1 "k8s.io/api/batch/v1"
 	coreV1 "k8s.io/api/core/v1"
 	nodeV1 "k8s.io/api/node/v1"
+	policyV1 "k8s.io/api/policy/v1" // Add this line to import the missing package
```
Comment (decleaver) [inline pkg/api/resources/cache.go:19]: can remove this comment, everything else looks good though!

Comment (TristanHoladay) [reply pkg/api/resources/cache.go:19]: naw, i want to let readers know exactly why i used that import .... :smile: Thanks, @decleaver

File: ui/src/lib/features/k8s/cluster-ops/pod-disruption-budgets/store.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  min_available: number | string
+  max_unavailable: number | string
+  current_healthy: number
+  desired_healthy: number
+}
+
+export type Columns = ColumnWrapper<Row>
+
+export function createStore(): ResourceStoreInterface<Resource, Row> {
+  // needs dense=true to get thhe min_available and max_unavailable fields from spec
```
Comment (jeff-mccoy) [inline ui/src/lib/features/k8s/cluster-ops/pod-disruption-budgets/store.ts:19]:
```suggestion
  // needs dense=true to get the min_available and max_unavailable fields from spec
```

**Review by jeff-mccoy — CHANGES_REQUESTED** [review]
tiny typo

**Comment by jeff-mccoy** [comment]
Some merge conflicts

**Review by jeff-mccoy — APPROVED** [review]

**Comment by Madeline-UX** [comment]
@TristanHoladay your screen is so big yet the screenshot was so small. I'm unable to see enough detail to do UX review lol but the shapes look about right. 🤣

**Comment by TristanHoladay** [comment]
is this better @Madeline-UX? i used dev tools to show it as 1920 x 1080 ...
Screenshot from 2024-07-27 19-26-30

**Pull request merged**